pkg/bgpv1/api/printers.go

```diff
@@ -59,6 +61,36 @@ func PrintBGPPeersTable(w *tabwriter.Writer, peers []*models.BgpPeer, printUptim
 	w.Flush()
 }
 
+// PrintBGPPeersCaps prints the capabilities of the provided BGP peers.
+func PrintBGPPeersCaps(w io.Writer, peers []*models.BgpPeer) {
+	if len(peers) == 0 {
+		fmt.Fprintf(w, "No BGP peer sessions found on this node\n")
+		return
+	}
+
+	for _, peer := range peers {
+		localCaps, err := decodeBgpCapabilities(peer.LocalCapabilities)
+		if err != nil {
+			fmt.Fprintf(w, "Could not decode local capabilities: %s\n", err)
```

Comment:
Errors should not be printed to the same output stream as the real output; errors should go to stderr.